app/src/components/integrations/containers/integrationInfoContainer/instancesSection/instancesSection.jsx

```diff
@@ -114,7 +114,7 @@ const messages = defineMessages({
 
 @connect(
   (state) => ({
-    projectId: projectIdSelector(state),
+    projectKey: urlProjectKeySelector(state),
```

Comment:
`projectId` is still present in prop types and is used inside the component.

Reply:
Ah, I see.
Neither one nor the second is used in the component. So I can just remove them.